### DISCUSSION TITLE
a few bugs

### DISCUSSION THREAD

**Discussion by zhubib1234** [javalent/obsidian-leaflet#64] [Q&A]
1. When "filter markers" closes all marker categories, click "show all markers" and the map will be gray instead of the map.
2. When "filter markers" selects some marker categories, click "show all markers", and then click "filter markers". The pop-up menu of "filter markers" only displays the previously selected marker categories.
3. When "filter markers" selects part of the marker categories, click "show all markers", close obsidian and open it again. The icons of other categories become "default". Is this bug related to the abnormal display of additional icons?

4. 3.10. X loading speed is slow,，The map loading speed is not as fast as 3.9. X.

**Comment by valentine195** [comment]
Thanks, I've rolled a few bugs related to the new filter markers feature into the next release. Unfortunately it probably won't be out until Monday.

> 3.10. X loading speed is slow,，The map loading speed is not as fast as 3.9. X.

Thanks for letting me know. The next release will prioritize loading the first image, then load the rest in the background, but I'll investigate as to what changed between 3.9.X and 3.10.X that could have caused this.

**Reply by zhubib1234** [reply]
Looking forward to the update.
